lib/inspec/profile_context.rb

```diff
@@ -91,7 +91,7 @@ def remove_rule(id)
     end
 
     def all_controls
-      ret = @rules.values
+      ret = @rules.values.reject(&:nil?)
```

Comment:
Seems like we could have also used `@rules.values.compact` here in this case? Even using reject will not have any implications, it's the same. It's just `compact` appears more neat. Just my opinion. What are your thoughts?

Reply:
That can be done. Thanks @Nik08